packages/connectivity-tests/test/localhost-run.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+set -x
+
+/opt/mongodb/bin/mongod --dbpath /var/mongodb/db &
+MONGOD_PID=$!
+FAILED=no
+
+sleep 10 # let mongod start up
+
+MONGOSH=/tmp/mongosh/packages/mongosh/bin/mongosh.js
```

Comment:
I noticed that in another test we are just using `mongosh` directly without a path and I'm not sure why it is different here 🤔

Reply:
This is because this is *inside* the docker container, where the `PATH` variable from outside (the one set in `all.sh`) doesn’t apply anymore :)

Reply:
Oooh, right, thanks for pointing that out, slipped my mind completely!